testData/annotation.semantic/ImplementExternInterface.hx

```diff
@@ -0,0 +1,7 @@
+class Test1 implements ExternInterfaceTest {
+  public function fooExtern(b:Int):Int return 2;
+}
+
+extern interface ExternInterfaceTest {
+  function fooExtern(b:Int):Int;
+}
```

Comment:
yes, if error any error annotation will appear in this case - test will fail, so this test check that no need to show error annotattion if we implements extern-interface.